xarray/tests/test_dataarray.py

```diff
@@ -2920,9 +2922,11 @@ def test_resample(self):
         actual = array.resample(time="24H").reduce(np.mean)
         assert_identical(expected, actual)
 
-        actual = array.resample(time="24H", loffset="-12H").mean()
```

Comment:
I don't think this is right unless we want to remove the `loffset` kwarg. 

If we keep it, then only `expected` needs to be changed. We'll need to update the code for `loffset` in `resample.py` to avoid the pandas warning.

If we want to remove it, I think we should remove it after `indexes` are more prominent and user-friendly.

Reply:
Yes, you're right. 

I was thinking we'd align our API with pandas' — unless we think we should define our own API — but then we actually need to make the changes rather than remove the tests!